third_party/lua/luadebug.lua

```diff
@@ -788,6 +788,19 @@ function dbg.call(f, ...)
     end, ...)
 end
 
+-- interactive console start
```

Comment:
Please start your comments with a capital letter and end with a dot. In the other places too.

Reply:
Done, thanks!

```diff
diff --git a/third_party/lua/luadebug.lua b/third_party/lua/luadebug.lua
index ce9e11440..4d6b00d71 100644
--- a/third_party/lua/luadebug.lua
+++ b/third_party/lua/luadebug.lua
@@ -788,10 +788,9 @@ function dbg.call(f, ...)
     end, ...)
 end
 
--- interactive console start
+-- Start an interactive debugging console.
 function dbg.start(path, ...)
-    -- all checks should be done in init.c
-    assert(path)
+    assert(path ~= nil and type(path) == 'string')
     assert(fio.stat(path) ~= nil)
 
     stack_inspect_offset = 0
```